storage/internal/benchmarks/client_pool.go

```diff
@@ -202,15 +201,15 @@ func initializeGRPCClient(ctx context.Context, writeBufferSize, readBufferSize i
 	if useDefaults {
 		clientMu.Lock()
 		os.Setenv("STORAGE_USE_GRPC", "true")
-		c, err := storage.NewClient(ctx, option.WithCredentialsFile(credentialsFile))
+		c, err := storage.NewClient(ctx)
```

Comment:
Do we no longer need to be able to customize the credentials source?

Reply:
Correct